src/ctr_logging.c

```diff
@@ -169,6 +173,22 @@ void configure_log_drivers(gchar **log_drivers, int64_t log_size_max_, int64_t l
 			syslog_identifier = g_strdup_printf("SYSLOG_IDENTIFIER=%s", tag);
 			syslog_identifier_len = strlen(syslog_identifier);
 		}
+		if (log_labels) {
+			// parse labels in the format "KEY=VALUE,KEY_2=VALUE_2,..."
+			container_labels_count = 1;
+			const char *ptr = log_labels;
+			while ((ptr = strchr(ptr, ',')) != NULL) {
+				container_labels_count++;
+				ptr++;
+			}
+			container_labels = g_alloca((sizeof(char *)) * container_labels_count);
+			container_labels_lengths = g_alloca((sizeof(int *)) * container_labels_count);
+			char *pair = strtok(log_labels, ",");
```

Comment:
`strtok` modifies the buffer, so `log_labels` will be modified.  The correct way to do it is to `g_strdup` the string before passing it to `strtok`.